### ISSUE TITLE
autocompletion of procedure fails

### ISSUE THREAD

**Issue by moertend** [vhdl-linter/vhdl-linter#494]
If writing the name of the procedure followed by brace open, there no autocompletion list.
![image](https://github.com/vhdl-linter/vhdl-linter/assets/30226425/780cf699-599b-422c-8a81-845429465f13)

**Comment by Nik-Sch** [comment]
I am pretty sure that this is the same problem as #493 as the file is currently not in a parsable state.

**Comment by g0t00** [comment]
This is kind of not a duplicate, as here it has to be an instantiation. The othercase could also be recovered to be an assignment. The great think is that here we need already context information to deduce that foo( is not actually going to be somethink like `foo(1 downto 0) <= "10";`
Would like to keep this open, as it for me is about recovering from incomplete procedure instantiation.

**Comment by Nik-Sch** [comment]
Yes you are correct, it is related but adds another level of recovery to it.
